### PR TITLE
Allow Hub to use Moneypenny

### PR DESCRIPTION
**Pull request by athornton**

### CODE REVIEW

File: charts/nublado/templates/jupyterhub/deployment.template.yml [hunk 1]
```diff
@@ -219,13 +226,11 @@ spec:
           persistentVolumeClaim:
            claimName: {{ template "nublado.fullname" . }}-hub-physpvc
 {{ end }}
-        - name: fs-mounts
-          configMap:
-            defaultMode: 0444
-            items:
-            - key: mountpoints.json
-              path: mountpoints.json
-            name: {{ template "nublado.fullname" . }}-fs-mounts
```
Comment (rra) [inline charts/nublado/templates/jupyterhub/deployment.template.yml:228]: Why are you deleting this? It seems unrelated.

Comment (athornton) [reply charts/nublado/templates/jupyterhub/deployment.template.yml:228]: That was a mistake.  Restored.

**Review by rra — APPROVED** [review]
